### PR TITLE
Zram: If "Device or resource busy", retry exec zramctl

### PR DESCRIPTION
**Pull request by robinzt**
Sometimes, systemd-swap produced error: **TypeError: stat: path should be string, bytes, os.PathLike or integer, not NoneType**

The cause is:
 If zramctl returned "failed to reset: Device or resource busy", systemd-swap just sleep 1 second but not retry exec zramctl.

So I change some code to retry exec zramctl when things happen.

The log:
```
systemd-swap[643]: INFO: Removing working directory...
systemd-swap[643]: INFO: Removing files in /var/lib/systemd-swap/swapfc/...
systemd-swap[643]: WARN: Combining zram with zswap/swapfc/swapd_auto_swapon can lead to LRU inversion and is strongly recommended against
systemd-swap[643]: INFO: Zram: check availability
systemd-swap[643]: INFO: Zram: not part of kernel, trying to find zram module
systemd-swap[643]: INFO: Zram: module successfully loaded
systemd-swap[643]: INFO: Zram: trying to initialize free device
systemd-swap[643]: Traceback (most recent call last):
systemd-swap[643]:   File "/usr/bin/systemd-swap", line 904, in <module>
systemd-swap[643]:     main()
systemd-swap[643]:   File "/usr/bin/systemd-swap", line 892, in main
systemd-swap[643]:     start()
systemd-swap[643]:   File "/usr/bin/systemd-swap", line 666, in start
systemd-swap[643]:     mode = os.stat(zram_dev).st_mode
systemd-swap[643]: TypeError: stat: path should be string, bytes, os.PathLike or integer, not NoneType

```

### CODE REVIEW

**Review by oold — APPROVED** [review]

File: systemd-swap [hunk 1]
```diff
@@ -662,7 +670,8 @@ def start() -> None:
                         "modprobe rule"
                     )
                 new_zram = read("/sys/class/zram-control/hot_add").rstrip()
-                info(f"Zram: success: new device /dev/zram{new_zram}")
+                zram_dev = f"/dev/zram{new_zram}"
+                info(f"Zram: success: new device {zram_dev}")
```
Comment (vilgotf) [inline systemd-swap:674]: Split this into another commit

**Review by vilgotf — APPROVED** [review]
LGTM
@oold merge with merge commit when your done

**Comment by vilgotf** [comment]
We should generally comment the code base better / refractor things as the python indentation syntax makes seeing when loops (etcetera) end hard

File: systemd-swap (hunk 1 above)
Comment (oold) [reply systemd-swap:674]: Will do.
